assets/style.css

```diff
@@ -0,0 +1,500 @@
+:root {
+  --indigo-dye: #013a63ff;
+  --light-blue: #a9d6e5ff;
+  --powder-blue: #caf0f8ff;
+}
+
+* {
+  margin: 0%;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  min-height: 100vh;
+  font-family: "Open Sans", sans-serif;
+  font-size: 10px;
+  line-height: 1.3;
+  background-color: var(--powder-blue);
+  color: var(--indigo-dye);
+}
+
+/* SECTION 1: Header */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 1rem 1rem 0px;
+  background-color: var(--indigo-dye);
+}
+
+.logo {
+  display: flex;
+  justify-content: end;
+  font-size: 3rem;
+  width: 500px;
+  padding: 10px;
+  background-color: var(--light-blue);
+  border-radius: 10px;
+  color: var(--indigo-dye);
+}
+
+.nav-bar-link {
+  display: flex;
+  flex-direction: row;
+}
+
+.nav-link {
+  margin-left: 80px;
+  align-self: center;
+  font-size: 2rem;
+}
+
+.link {
+  text-decoration: underline 0.7rem;
+  color: var(--light-blue);
+  text-decoration: underline;
+}
+
+/* SECTION 2: Banner */
+.banner {
+  border: 0.5rem var(--indigo-dye) solid;
+  height: 13rem;
+  background-image: url("../assets/images/pawel-czerwinski-eUVI3DQrTsE-unsplash\ \(1\).jpg");
+  background-position: top;
+  background-size: cover;
+  position: relative;
+}
+
+.name {
+  text-decoration: solid;
+  font-size: 4.5rem;
+  text-transform: uppercase;
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  animation-name: coloranimation;
+  animation-duration: 4s;
+  animation-iteration-count: infinite;
+}
```

Comment:
As mentioned on the feedback, I really appreciate the colour changing text. It's a good bit of code, but it is difficult to read on the main site. I think you'd benefit more with one solid colour with a border or a shadow. Entirely optional though!